calicoctl/calico_ctl/node.py

```diff
@@ -612,8 +612,16 @@ def node_remove(remove_endpoints):
                         "in the normal way.")
         sys.exit(1)
 
+    # Remove the veths, and release all IPs associated with the endpoint.
+    ips = set()
     for endpoint in endpoints:
         remove_veth(endpoint.name)
+        ips |= {net.ip for net in endpoint.ipv4_nets}
+        ips |= {net.ip for net in endpoint.ipv6_nets}
+    client.release_ips(ips)
+
+    # Remove the IPAM host data.
+    client.remove_ipam_host(hostname)
```

Comment:
Watch for conflicts with master here, you'll need to update this `hostname` value to `host_to_remove`.